contracts/pcv/curve/CurvePCVDepositPlainPool.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "../../Constants.sol";
+import "../PCVDeposit.sol";
+import "./ICurveStableSwap3.sol";
+
+/// @title CurvePCVDepositPlainPool: implementation for a PCVDeposit that deploys
+/// liquidity on Curve, in a plain pool (3 stable assets).
+/// @author Fei Protocol
+contract CurvePCVDepositPlainPool is PCVDeposit {
+
+    // ------------------ Properties -------------------------------------------
+
+    /// @notice maximum slippage accepted during deposit / withdraw, expressed
+    /// in basis points (100% = 10_000).
+    uint256 public maxSlippageBasisPoints;
+
+    /// @notice The Curve pool to deposit in
+    address public curvePool;
```

Comment:
This should be a type of `ICurveStableSwap3 ` instead of address as many other places in the codebase it is casted to a `ICurvePool` or `ICurveStableSwap3` and the ICurvePool interface is inherited in the ICurveStableSwap3 interface.